prepare/cards/numeric_nlg.py

```diff
@@ -0,0 +1,56 @@
+from unitxt.blocks import (
+    AddFields,
+    LoadHF,
+    MapHTMLTableToJSON,
+    RenameFields,
+    SerializeTableAsMarkdown,
+    Task,
+    TaskCard,
+    TemplatesList,
+)
+from unitxt.catalog import add_to_catalog
+from unitxt.splitters import SplitRandomMix
+from unitxt.templates import InputOutputTemplate
+from unitxt.test_utils.card import test_card
+
+card = TaskCard(
+    loader=LoadHF(path="kasnerz/numericnlg"),  # TODO: load from github repo
+    preprocess_steps=[
+        SplitRandomMix({"train": "train", "validation": "validation", "test": "test"}),
+        AddFields(fields={"type_of_input": "table", "type_of_output": "text"}),
+        MapHTMLTableToJSON(field_to_field=[["table_html_clean", "table_out"]]),
+        SerializeTableAsMarkdown(field_to_field=[["table_out", "input"]]),
+        RenameFields(field_to_field={"description": "output"}),
+    ],
+    task=Task(
+        inputs={"input": "str", "type_of_input": "str", "type_of_output": "str"},
+        outputs={"output": "str"},
+        prediction_type="str",
+        metrics=[
+            "metrics.bleu",
+            "metrics.rouge",
+            "metrics.bert_score.bert_base_uncased",
+            "metrics.meteor",
+        ],
+        # add PARENT [we leave it for now, as it doesn't have to be the chosen metric]
+        augmentable_inputs=["input"],
+    ),
```

Comment:
```suggestion
    task="tasks.generation[metrics=[metrics.bleu,metrics.rouge,metrics.bert_score.bert_base_uncased,metrics.meteor]",
```
This will help us to index the task in UnitxtExplore demo on the website.